atlite/cutout.py

```diff
@@ -60,8 +61,8 @@ def __init__(self, name=None, data=None, cutout_dir=None, **cutoutparams):
         if dirname:
             cutout_dir = dirname
         elif cutout_dir is None:
-            if config.cutout_dir:
-                cutout_dir = utils.construct_filepath(config.cutout_dir)
+            if config.config.cutout_dir:
+                cutout_dir = config.config.cutout_dir
```

Comment:
```suggestion
                cutout_dir = self.config.cutout_dir
```

Here and in the line above it should be `self.config.cutout_dir`.

However:
Making the config part of the cutout should make the attribute `cutout.cutout_dir` obsolete.
We can keep it in the descriptor, but we should at this point - if not set - `self.config.cutout_dir = cutout_dir`
and substitute subsequent calls accordingly.

Reply:
Good idea,

I cleaned up all uses of `cutout_dir` and removed the variable from the constructor's call signature. Instead I introduced a `Cutout.cutout_path` which points to the netcdf file or old-style directory the cutout is based on.